packages/calcite-components/src/components/filter/filter.tsx

```diff
@@ -81,6 +81,16 @@ export class Filter
    */
   @Prop({ mutable: true }) filteredItems: object[] = [];
 
+  /**
+   * Specifies the fields to match against when filtering.
```

Comment:
Could you add a note about this only applying when `value` is an object and that all fields are matched if not set? +@geospatialem @DitwanP for an editorial look-see. 👀

Reply:
@geospatialem @DitwanP can you comment on...

> Specifies the fields to match against when filtering. This will only apply when `value` is an object. If not set, all fields will be matched.

Reply:
Looks good to me, clear and descriptive.